<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoControlSimple</class>
 <widget class="QWidget" name="VideoControlSimple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoControlWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="_firstButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="_lastButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frame Number:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="_frameOut"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>193</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_firstButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoControlSimple</receiver>
   <slot>first_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_lastButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoControlSimple</receiver>
   <slot>last_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
